### ISSUE TITLE
Increase user preview size on large screens

### ISSUE THREAD

**Issue by jlnr** [elementary/greeter#303]
I am using a 38" ultrawide for development, and the new greeter feels out of place because it's mostly gray. It will get even lonelier without the Libvirt Guest from #296.

![IMG_6416](https://user-images.githubusercontent.com/70772/60383332-98f8cb80-9a6f-11e9-9bf2-d7b819042265.jpeg)

Would it be reasonable to increase the size of the user boxes in relation to the available screen size? That way I could see more of my pretty wallpaper.

**Comment by cassidyjames** [comment]
Yeah, we should consider setting the boxes as a percent of the display size or something. I just don't know what would make sense visually _and_ be easy to implement.

**Comment by CallieG** [comment]
I'm here to second, third and fourth this request.   Can we also make this part of the Universal Access thing?   I can certainly "see" the greeter, but I wear glasses, and it's even a bit small, on a laptop that I run in 1600x900.

**Comment by BAProductions** [comment]
I agree
